Assets/HoloToolkit-Examples/ColorPicker/GazeableColorPicker.cs

```diff
@@ -50,5 +36,30 @@ void UpdatePickedColor(PickedColorCallback cb)
             Color col = texture.GetPixel((int)pixelUV.x, (int)pixelUV.y);
             cb.Invoke(col);
         }
+
+        public void OnFocusEnter()
+        {
+            gazing = true;
+        }
+
+        public void OnFocusExit()
+        {
+            gazing = false;
+        }
+
+        public void OnInputUp(InputEventData eventData)
```

Comment:
It could, but ISourceStateHandler is already used for visibility. I think it'd make sense to do this:

ISourceStateHandler: source up/down
ISourceVisibilityHandler: source detected / lost

I can look at making that change tomorrow, it shouldn't be too bad.